src/codegen/sys/functions.rs

```diff
@@ -155,6 +156,18 @@ pub fn generate_other_funcs(
     generate_object_funcs(w, env, obj, "Other functions", INTERN, functions)
 }
 
+fn generate_cfg_configure(
+    w: &mut Write,
+    functions: &Vec<&Function>,
+    commented: bool,
+)-> Result<()> {
+    let with_cfg_condition = functions.iter().find(|f| f.cfg_condition.is_some());
+    if let Some(info) = with_cfg_condition {
+        cfg_condition(w,  &info.cfg_condition, commented, 1)?;
+    }
```

Comment:
Please use something like next here:
```
    let cfg_condition = configured_functions
        .iter()
        .filter_map(|f| f.cfg_condition.clone())
        .next();
    cfg_condition(w,  &cfg_condition, commented, 1)?;
```

Reply:
Maybe without `clone`

Reply:
without clone(): error[E0507]: cannot move out of borrowed content

Reply:
Hm. `clone` can't be removed, so better leave as is, sorry.